codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright 2015 The Netty Project
```

Comment:
2016 ... man bet you will be happy when we can get this in and you don't have to keep rebasing :smile: